#5144 Extend development environment documentation (#7331)
hzi-braunschweig/SORMAS-Project: docs/DEVELOPMENT_ENVIRONMENT.md

  - Enter `http://localhost:6080/sormas-ui` into the `URL` field
  - Make sure that the correct JRE is specified (your Java 11 JDK)
  - Enter the path to the SORMAS domain and the credentials that you've specified when setting up the server

>   - Open the `Deployment` tab and add the artifacts `sormas-ear`, `sormas-rest` and `sormas-ui` (make sure the respect the order as specified here as there are depdendencies between artifacts at startup)

Reviewer: ```suggestion
  - Open the `Deployment` tab and add the artifacts `sormas-ear`, `sormas-rest` and `sormas-ui` (make sure to respect this order as there are dependencies between artifacts at startup)
```

- Either run `mvn install` on the `sormas-base` project or execute the `install [default]` Ant script (this needs a Maven installation on your system with the M2_HOME variable set)
- Execute the `deploy-serverlibs` Ant script
- Highlight all Eclipse projects and choose `Maven -> Update Project` from the right-click menu; perform the update for all projects

> - Start the Glassfish server and deploy `sormas-ear`, `sormas-rest` and `sormas-ui` by dragging the respective projects onto it, or use the `Add and Remove...` function by right-clicking on the server (make sure the respect the order as specified here as there are depdendencies between artifacts at startup)

Reviewer: ```suggestion
- Start the Glassfish server and deploy `sormas-ear`, `sormas-rest` and `sormas-ui` by dragging the respective projects onto it, or use the `Add and Remove...` function by right-clicking on the server (make sure to respect this order as there are depdendencies between artifacts at startup)
```
